bindings/core/Cargo.toml

```diff
@@ -34,3 +34,4 @@ participation = [ "iota-sdk/participation" ]
 rocksdb = [ "iota-sdk/rocksdb" ]
 storage = [ "iota-sdk/storage" ]
 stronghold = [ "iota-sdk/stronghold" ]
+crypto_rand = [ "iota-crypto/random" ]
```

Comment:
Why is this now separated?
Without this feature enabled I get
```
error[E0433]: failed to resolve: could not find `rand` in `utils`
  --> sdk/src/client/utils.rs:56:12
   |
56 |     utils::rand::fill(&mut entropy)?;
   |            ^^^^ could not find `rand` in `utils`
```

Reply:
It was included by one of the other features we used but the crypto update changed that so now we need it manually. But it cannot be used in the core manifest because it does not support `no_std`.

Reply:
Why don't just add it to the client feature?

Reply:
No, because wasm uses client but cannot use this.